compute/cloud-client/src/test/java/compute/Util.java

```diff
@@ -79,52 +105,24 @@ && isCreatedBeforeThresholdTime(template.getCreationTimestamp())
   public static void cleanUpExistingInstances(String prefixToDelete, String projectId,
       String instanceZone)
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
-    for (Map.Entry<String, InstancesScopedList> instanceGroup : listFilteredInstances(
-        projectId, prefixToDelete).iterateAll()) {
-      for (Instance instance : instanceGroup.getValue().getInstancesList()) {
-        if (!instance.hasCreationTimestamp()) {
-          continue;
+    try (InstancesClient instancesClient = InstancesClient.create()) {
+      for (Instance instance : instancesClient.list(projectId, instanceZone).iterateAll()) {
+        if (instance.getDeletionProtection()
+            && isCreatedBeforeThresholdTime(instance.getCreationTimestamp())) {
+          SetDeleteProtection.setDeleteProtection(
+              projectId, instanceZone, instance.getName(), false);
         }
-        if (instance.getName().contains(prefixToDelete)
-            && isCreatedBeforeThresholdTime(instance.getCreationTimestamp())
-            && instance.getStatus().equalsIgnoreCase(Status.RUNNING.toString())) {
+        if (containPrefixToDeleteAndZone(instance, prefixToDelete, instanceZone)
+            && isCreatedBeforeThresholdTime(instance.getCreationTimestamp())) {
           DeleteInstance.deleteInstance(projectId, instanceZone, instance.getName());
         }
       }
     }
   }
 
-  public static AggregatedListPagedResponse listFilteredInstances(
-      String project, String instanceNamePrefix) throws IOException {
-    try (InstancesClient instancesClient = InstancesClient.create()) {
-
-      AggregatedListInstancesRequest aggregatedListInstancesRequest = AggregatedListInstancesRequest
-          .newBuilder()
-          .setProject(project)
-          .setFilter(String.format("name:%s", instanceNamePrefix))
-          .build();
-
-      return instancesClient
-          .aggregatedList(aggregatedListInstancesRequest);
-    }
-  }
-
-  public static ListPagedResponse listFilteredInstanceTemplates(
-      String projectId, String instanceTemplatePrefix) throws IOException {
-    try (InstanceTemplatesClient instanceTemplatesClient = InstanceTemplatesClient.create()) {
-      ListInstanceTemplatesRequest listInstanceTemplatesRequest =
-          ListInstanceTemplatesRequest.newBuilder()
-              .setProject(projectId)
-              .setFilter(String.format("name:%s", instanceTemplatePrefix))
-              .build();
-
-      return instanceTemplatesClient.list(listInstanceTemplatesRequest);
-    }
-  }
-
   public static boolean isCreatedBeforeThresholdTime(String timestamp) {
     return OffsetDateTime.parse(timestamp).toInstant()
-        .isBefore(Instant.now().minus(DELETION_THRESHOLD_TIME_HOURS, ChronoUnit.HOURS));
+        .isBefore(Instant.now().minus(DELETION_THRESHOLD_TIME_HOURS, ChronoUnit.MINUTES));
```

Comment:
Changed to MINUTES to have possibility to delete resources for failed tests. Will be replaced to HOURS when samples for reservation is done

Reply:
Okay, please update the variable name to denote minutes.